README.md

```diff
@@ -22,78 +22,55 @@ by some organizations to back user interfaces for KubeRay resource management.
 
 * **KubeRay CLI**: KubeRay CLI provides the ability to manage KubeRay resources through command-line interface.
 
-## KubeRay ecosystem
-
-* [AWS Application Load Balancer](docs/guidance/ingress.md)
-* [Nginx](docs/guidance/ingress.md)
-* [Prometheus and Grafana](docs/guidance/prometheus-grafana.md)
-* [Volcano](docs/guidance/volcano-integration.md)
-* [MCAD](docs/guidance/kuberay-with-MCAD.md)
-* [Kubeflow](docs/guidance/kubeflow-integration.md)
-
-## Blogs
-
-* [A cloud-native, open-source stack for accelerating foundation model innovation](https://research.ibm.com/blog/openshift-foundation-model-stack) IBM (May 9, 2023).
-* [AI/ML Models Batch Training at Scale with Open Data Hub](https://cloud.redhat.com/blog/ai/ml-models-batch-training-at-scale-with-open-data-hub) Red Hat (May 15, 2023).
-
 ## Documentation
 
-You can view detailed documentation and guides at [https://ray-project.github.io/kuberay/](https://ray-project.github.io/kuberay/).
-
-We also recommend checking out the official Ray guides for deploying on Kubernetes at [https://docs.ray.io/en/latest/cluster/kubernetes/index.html](https://docs.ray.io/en/latest/cluster/kubernetes/index.html).
+From September 2023, most KubeRay documentation will be hosted on [Ray documentation](https://docs.ray.io/en/latest/cluster/kubernetes/index.html).
```

Comment:
```suggestion
From September 2023, KubeRay documentation will be hosted on the [Ray documentation](https://docs.ray.io/en/latest/cluster/kubernetes/index.html).
```

Ideally we can also give the reader some idea of which documents remain on the KubeRay docs, otherwise they might be curious/confused.

Reply:
Updated https://github.com/ray-project/kuberay/pull/1433/commits/f443c26721f8af7ca9e6b8cb557012b96a432fc7